### DISCUSSION TITLE
Is this a mistake?

### DISCUSSION THREAD

**Discussion by mukhammart** [evildmp/diataxis-documentation-framework#105] [General]
![image](https://github.com/evildmp/diataxis-documentation-framework/assets/50018996/d9fa28b6-450a-4d58-ac59-de62d2f59eb8)
I gues there should be "be", between words _should_ and _read_

**Comment by waldyrious** [comment]
It's not a mistake — that is one of the valid ways to use the verb "read". See the entry prefixed "(ergative)" in <https://en.wiktionary.org/wiki/read#Verb>.
